core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/distinct.kt

```diff
@@ -23,45 +29,60 @@ import kotlin.reflect.KProperty
 // region DataFrame
 
 /**
- * ## The Distinct Operation
+ * {@get [DESCRIPTION] Removes duplicated rows based on {@get [PHRASE_ENDING]}}.
  *
- * It removes duplicated rows based on {@get PHRASE_ENDING}.
+ * The [rows][DataRow] in the resulting [DataFrame] are in the same order
+ * as they were in the original [DataFrame].
  *
- * __NOTE:__ The rows in the resulting [DataFrame] are in the same order as they were in the original [DataFrame].
+ * See also {@get [FUNCTION] [distinctBy] that removes duplicated rows based on the specified columns
+ * and keeps all the columns in the resulting [DataFrame].}
  *
- * {@get [DISTINCT_PARAM] @param [columns]
- * The names of the columns to consider for evaluating distinct rows.}
+ * @include [SelectingColumns.ColumnGroupsAndNestedColumnsMention]
+ *
+ * See [Selecting Columns][SelectSelectingOptions].
+ *
+ * For more information:
  *
- * @return A new DataFrame containing only distinct rows.
+ * {@include [DocumentationUrls.Distinct]}
  *
- * @see [Selecting Columns][SelectSelectingOptions].
- * @see {@include [DocumentationUrls.Distinct]}
+ * {@include [DocumentationUrls.DistinctBy]}
+ *
+ * {@get [DISTINCT_PARAM]}
+ *
+ * @return {@get [DISTINCT_RETURN] A new [DataFrame] containing only distinct rows.}
  */
 @ExcludeFromSources
 @Suppress("ClassName")
 private interface DistinctDocs {
     interface DISTINCT_PARAM
+
+    interface DISTINCT_RETURN
+
+    interface DESCRIPTION
+
+    interface PHRASE_ENDING
+
+    interface FUNCTION
```

Comment:
I usually write a tiny comment (no KDoc) next to each parameter, to clarify what they mean